lib/scan/detect_values.rb

```diff
@@ -23,6 +23,22 @@ def self.set_additional_default_values
       return config
     end
 
+    def self.filter_simulators(simulators, deployment_target)
+      # Filter out any simulators that are not the same major and minor version of our deployment target
+      if deployment_target.to_s.length > 0
+        components = deployment_target.split('.')
+        major = (components.length >= 1) ? components[0] : 0
+        minor = (components.length >= 2) ? components[1] : 0
+        simulators = simulators.select do |s|
+          sim_components = s.ios_version.split('.')
+          sim_major = (sim_components.length >= 1) ? sim_components[0] : 0
+          sim_minor = (sim_components.length >= 2) ? sim_components[1] : 0
+          (sim_major > major) || ((sim_minor >= minor) && (sim_major == major))
+        end
```

Comment:
Ah, cool!  I'll update using this. This bug is referenced from a few other issues, so I think it is worth getting this fixed relatively soon. Thanks for feedback!